ui/src/app/archived-workflows/components/archived-workflow-list/archived-workflow-list.tsx

```diff
@@ -72,6 +74,14 @@ export class ArchivedWorkflowList extends BasePage<RouteComponentProps<any>, Sta
         );
     }
 
+    public componentDidUpdate(prevProps: Readonly<RouteComponentProps<any, StaticContext>>, prevState: Readonly<State>, snapshot?: any): void {
```

Comment:
perhaps we can remove these parameters since they are unused?

Reply:
Yes I can do that